moveit_planners/chomp/chomp_interface/include/chomp_interface/chomp_planning_context.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * chomp_planning_context.h
+ *
+ *  Created on: 27-Jul-2016
+ *      Author: ace
```

Comment:
They should be attributed to ace, but if you put a lot of refactoring work into it I think you can add your name also